### PR TITLE
minor test improvements

### PR DESCRIPTION
**Pull request by oscardssmith**
These are a few fixes to CI that I found while trying to give Diffractor an interface.

### CODE REVIEW

File: src/codegen/reverse.jl [hunk 1]
```diff
@@ -465,7 +465,8 @@ function diffract_ir!(ir, ci, meth, sparams::Core.SimpleVector, nargs::Int, N::I
             elseif isexpr(stmt, :static_parameter)
                 fwds[i] = ZeroTangent()
             elseif isa(stmt, Union{GotoNode, GotoIfNot})
-                return :(error("Control flow support not fully implemented yet for higher-order reverse mode (TODO)"))
+                #@show stmt
+                error("Control flow support not fully implemented yet for higher-order reverse mode (TODO)")
```
Comment (oxinabox) [inline src/codegen/reverse.jl:469]: Why do we want to move this error to compile time,  rather then runtime?

Comment (oscardssmith) [reply src/codegen/reverse.jl:469]: well if we want it at run time, we need to generate ir rather than an expression

Comment (oxinabox) [reply src/codegen/reverse.jl:469]: oh true

**Review by oxinabox — APPROVED** [review]

**Comment by oscardssmith** [comment]
re-running CI now that Julia master is fixed.

**Pull request merged**